srcdoc/configure/proxy_directives.mt

```diff
@@ -319,6 +319,22 @@ But if the backend server has enough concurrency, <code>proxy.max-buffer-size</c
 </p>
 ? })
 
+<?
+$ctx->{directive}->(
+    name     => "proxy.max-pipes",
```

Comment:
Thank you for the doc! The explanation looks perfect.

Regarding the name, please refer to my other comment.

Reply:
Thanks -- I'll resolve this conversation. I've added a commit to change the name as suggested.